device/src/region/mod.rs

```diff
@@ -188,8 +189,11 @@ impl Configuration {
 
     pub(crate) fn process_join_accept<T: core::convert::AsRef<[u8]>, C>(
         &mut self,
+        mac: &mut crate::Mac,
```

Comment:
nit: I have a slight preference of having `mac` as the second parameter, as my perception is that `join_accept` is usually the first parameter when provided and if that's the case, it's worth keeping the consistency.